kpops/component_handlers/helm_wrapper/helm_diff.py

```diff
@@ -16,7 +16,7 @@ def __init__(self, config: HelmDiffConfig) -> None:
     def calculate_changes(
         current_release: Iterable[HelmTemplate],
         new_release: Iterable[HelmTemplate],
-    ) -> Iterator[Change[KubernetesManifest, KubernetesManifest]]:
+    ) -> Iterator[Change[dict[str, Any], dict[str, Any]]]:
```

Comment:
do we really have to dump the dict here already? Can't we return the Pydantic model and let the caller decide what to do with it? Or is that impractical?

Reply:
The problem with returning the model is that we have no value for an empty state (`{}` or `KubernetesManifest()`) because the fields `api_version`, `kind`, `metaddata` are mandator.